shannonlal/mcp-postman
# feat: enable build files

### Task List

- [x] You have included code examples or test cases demonstrating your changes
- [x] You have completed local testing with Postman collections

### Description

This PR enables build files by removing the `build` directory from `.gitignore` and committing the built files. This change allows users to directly use the MCP server without having to build it locally first, improving the user experience and reducing setup complexity.

Current behavior: Users need to build the project locally before using the MCP server.
Updated behavior: Built files are included in the repository, allowing immediate use after installation.

### Testing Notes

To test this PR:

1. Clone the repository
2. Configure the MCP server in your Cline settings:
```json
{
  "mcpServers": {
    "postman-runner": {
      "command": "node",
      "args": ["/absolute/path/to/mcp-postman/build/index.js"]
    }
  }
}
```
3. Test with the example Postman collection

Expected behavior: The MCP server should work immediately without requiring a build step.

### Test Results

```
✓ MCP server starts successfully
✓ Postman collection runs without build step
✓ All integration tests passing
```

### Additional Context

This change simplifies the setup process for users by including pre-built files in the repository. The build files are now tracked in git and will be updated with each release, ensuring consistency between the source code and the distributed version.
